tools/installpsh-debian.sh

```diff
@@ -176,12 +176,12 @@ fi
 case $DISTRIB_ID in
     ubuntu|linuxmint)
         case $DISTRIB_RELEASE in
-            18.04|16.10|16.04|15.10|14.04)
+            20.04|18.04|16.10|16.04|15.10|14.04)
```

Comment:
I think we should remove unsupported versions.

Reply:
@iSazonov Should I do that in the context of this PR or as a separate one? Also maybe that would be a breaking change for existing CI/CD deployments?

Reply:
@TravisEz13 Could you help please? What versions could we remove without breaking CIs?

Reply:
These are the distributions most common to Azure Devops and Github Actions, so the short term ".10" ubuntu releases could probably be dropped, I'd be a little wary of 14.04 though as it may have wide deployment in onprem CI environments like Jenkins, etc.
https://github.com/actions/virtual-environments#available-environments

Reply:
Rebased to master for fixed test. 
@iSazonov @TravisEz13 do you want me to drop the unsupported ".04" releases or should that be a separate PR (if at all)?

My opinion is leave it, don't create a potential breaking change but if someone does a net-new install to these and asks for support just direct them to the supported platform document.

Reply:
The release process is completely under the control of MSFT team, and they depend on internal (cloud) projects. So we are waiting for their conclusion.
Current OS support document for .Net 6 https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0-supported-os.md
So we could safely remove 16.10 and 15.10.

Reply:
I'm fine with a separate PR.  PowerShell policy is that the .10 releases are community supported.  We don't build packages for those releases.